tests/functions/test_process_text.py

```diff
@@ -4,46 +4,174 @@
 from pandas.testing import assert_frame_equal
 
 
-def test_str_split():
-    "Test wrapper for Pandas ``.str.split()`` method."
-
-    df = pd.DataFrame(
+@pytest.fixture
+def test_df():
```

Comment:
Minor nitpick, which I think might affect pytest usage: fixtures commonly don't have a `test_` prefix. Could we call this fixture `process_text_df`, and rename them accordingly below please?

Reply:
Thanks @ericmjl i will work on the suggestions